internal/node/node.bzl

```diff
@@ -591,7 +593,8 @@ Predefined genrule variables are not supported in this context.
         default = Label("//internal/node:loader.js"),
         allow_single_file = True,
     ),
-    "_node": attr.label(
+    "node": attr.label(
+        # the default needs to change and it shouldn't be private so that it can be set by the user or a macro
```

Comment:
"shouldn't be private" is now out-of-date right?

Reply:
I think https://github.com/bazelbuild/rules_nodejs/issues/3124 is where we'll be forced to change the default

Reply:
yes that is out of date, this should be address in a follow up pr that helps remove the reference to @nodejs